### PR TITLE
compataibility for php 8.2 to stop error Deprecated: Creation of dyna…

### PR DESCRIPTION
**Pull request by pcmad**
…mic property

file change see attached Zend_Controller_Action

example of code below for controller for ajax json output 
class TestController extends Zend_Controller_Action
{
	public function init ()
	{
 $contextSwitch = $this->_helper->getHelper('contextSwitch');

		$contextSwitch->addActionContext('ajax-attribute-checker', 'json')->initContext('json');

}
public function ajaxAttributeCheckerAction ()
	{
	
	
			$this->view->return = array('yay'=>true);
		}
	}
}

### CODE REVIEW

**Comment by holtkamp** [comment]
@develart-projects why not use a typed property like:

```php
public array $contexts;
```

?

File: library/Zend/Controller/Action.php [hunk 1]
```diff
@@ -91,6 +91,12 @@ abstract class Zend_Controller_Action implements Zend_Controller_Action_Interfac
      */
     public $view;
 
+    /**
+     * compataibility for php 8.2 to stop error Deprecated: Creation of dynamic property 
+     * @var object 
+     */
+    public  $contexts = null;
```
Comment (holtkamp) [inline library/Zend/Controller/Action.php:98]: why not use a typed property?

Comment (develart-projects) [reply library/Zend/Controller/Action.php:98]: Because it PHP.

Comment (holtkamp) [reply library/Zend/Controller/Action.php:98]: The PHPDoc `@var object` now suggests it is an `object`, which it is not, it is an `array`

**Pull request merged**